torch/csrc/api/src/nn/modules/loss.cpp

```diff
@@ -49,5 +49,25 @@ Tensor CosineEmbeddingLossImpl::forward(
   return F::cosine_embedding_loss(input1, input2, target, options);
 }
 
+// ============================================================================
+
+TripletMarginLossImpl::TripletMarginLossImpl(
+    const TripletMarginLossOptions& options_)
+    : options(options_) {}
+
+void TripletMarginLossImpl::pretty_print(std::ostream& stream) const {
+  stream << "torch::nn::TripletMarginLoss(p=" << options.p() << 
+            ", margin=" << options.margin() <<
+            ", weight=" << options.swap() <<
```

Comment:
Do you mean to write `swap=` instead of `weight=`? (We will need `std::boolalpha` for printing boolean value in C++.)